### DISCUSSION TITLE
[JS] Is there a way to capture a keyboard shortcut and decorate its action

### DISCUSSION THREAD

**Discussion by aledeg** [FreshRSS/FreshRSS#5507] [Q&A]
I want to use shortcuts to navigate through articles and when an article contains pictures, I want to navigate through the pictures. I do not want to replace completely the shortcut but enhance it when certain conditions are met.

**Comment by Frenzie** [comment]
Sure, just say something like onkeypress j, nextCallback() and define nextCallBack = nextCallBackDefault.

Then in your extension/userscript you could nextCallBack = someCustomFunction to add a bunch of logic that calls the default if the image condition isn't met.

**Reply by aledeg** [reply]
Thank you, I'll look into that.
